blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -212,3 +212,8 @@ def test_join_type_promotion(sqla, sqlb):
     result = set(map(tuple, compute(expr, {t: sqla, s: sqlb}).execute().fetchall()))
     expected = set([(1, 'a', 'a'), (1, None, 'a')])
     assert result == expected
+
+
+def test_interactive_len(sql):
+    t = Data(sql)
+    assert len(t) == odo(t.count(), int)
```

Comment:
oh, I didn't realize `__int__` was the same. I like the clarity of `odo` here; however, I don't mind promoting this pattern.